### PR TITLE
explicitly wire the aggregation information and use consistent client CA

### PR DESCRIPTION
**Pull request by deads2k**
This prevents variation in clusters during bring up and ensures that our operator has precise (and well logged) control over what authentication information is used.

### CODE REVIEW

File: pkg/cmd/openshift-apiserver/openshiftapiserver/config.go [hunk 1]
```diff
@@ -125,6 +125,15 @@ func NewOpenshiftAPIConfig(config *openshiftcontrolplanev1.OpenShiftAPIServerCon
 		return nil, err
 	}
 	authenticationOptions := genericapiserveroptions.NewDelegatingAuthenticationOptions()
+	authenticationOptions.ClientCert.ClientCA = config.ServingInfo.ClientCA
```
Comment (sttts) [inline pkg/cmd/openshift-apiserver/openshiftapiserver/config.go]: was this not wired?!

**Comment by sttts** [comment]
lgtm

File: pkg/cmd/openshift-apiserver/openshiftapiserver/config.go (hunk 1 above)
Comment (deads2k) [reply pkg/cmd/openshift-apiserver/openshiftapiserver/config.go]: > was this not wired?!

nope.  The server was configured with one and a different one was sent for authentication.  I think we made them both agree eventually.

**Comment by deads2k** [comment]
now with the actual bump

**Comment by deads2k** [comment]
/retest


look like kube-controller-manager (which this pull didn't touch).  If I pass or mostly pass aws, I'm going to merge this to have a go at completing the wiring.

**Comment by deads2k** [comment]
I've gotten green on the individual tests and I want to hit the hour.  I'm going for it.

**Pull request merged**